lightning/src/ln/offers_tests.rs

```diff
@@ -662,11 +662,7 @@ fn creates_and_pays_for_refund_using_two_hop_blinded_path() {
 	expect_recent_payment!(david, RecentPaymentDetails::Fulfilled, payment_id);
 }
 
-/// Checks that an offer can be paid through a one-hop blinded path and that ephemeral pubkeys are
-/// used rather than exposing a node's pubkey. However, the node's pubkey is still used as the
-/// introduction node of the blinded path.
-#[test]
-fn creates_and_pays_for_offer_using_one_hop_blinded_path() {
+fn do_creates_and_pays_for_offer_using_one_hop_blinded_path(with_message_retry: bool) {
```

Comment:
Let's make a dedicate test for this behavior rather than modifying an existing test.

Reply:
Done! [fa5cf60](https://github.com/lightningdevkit/rust-lightning/commit/fa5cf6003f56940682f578688cd33c68838086e9)
Thanks!